src/favorites.rs

```diff
@@ -35,14 +35,13 @@ pub(crate) fn list_favorites(args: &Args) -> Result<()> {
 }
 
 pub(crate) fn resolve_favorite(args: &mut Args) -> Result<()> {
-    if args.git.is_some() {
+    if args.git.is_some() || args.folder.is_some() {
         return Ok(());
     }
 
-    let favorite_name = args
-        .favorite
-        .as_ref()
-        .ok_or_else(|| anyhow!("Please specify either --git option, or a predefined favorite"))?;
+    let favorite_name = args.favorite.as_ref().ok_or_else(|| {
```

Comment:
instead of using `ok_or_else` to return the error, you could use `.context("Please specify either --git, --folder, or a predefined favorite")?` to resolve the result or yield the error

Reply:
I just keep forgetting 😆